e2e_test/source/cdc/cdc.share_stream.slt

```diff
@@ -268,3 +268,62 @@ SELECT * from person_new order by id;
 1100 noris ypl@qbxfg.com 1864 2539 enne
 1101 white myc@xpmpe.com 8157 6974 se
 1102 spencer wip@dkaap.com 9481 6270 angeles
+
+statement ok
+CREATE TABLE numeric_to_rw_int256_shared (
+    id int,
+    num rw_int256,
+    PRIMARY KEY (id)
+) FROM pg_source TABLE 'public.numeric_table';
+
+statement ok
+CREATE TABLE numeric_to_varchar_shared (
+    id int,
+    num varchar,
+    PRIMARY KEY (id)
+) FROM pg_source TABLE 'public.numeric_table';
+
+
+system ok
+psql -c "
+insert into numeric_table values(102, 57896044618658097711785492504343953926634992332820282019728792003956564819967);
+--- 2^255
+insert into numeric_table values(103, 57896044618658097711785492504343953926634992332820282019728792003956564819968);
+--- 2^256
+insert into numeric_table values(104, 115792089237316195423570985008687907853269984665640564039457584007913129639936);
+insert into numeric_table values(105, 115792089237316195423570985008687907853269984665640564039457584007913129639936.555555);
+"
+
+sleep 3s
+
+query III rowsort
+select * from numeric_to_varchar_shared
```

Comment:
I think you should use `order by` to make the output deterministic.

Reply:
update